### DISCUSSION TITLE
Are there really only 10 people working on 2k issues?

### DISCUSSION THREAD

**Discussion by hutterm** [dotnet/maui#11568] [General]
so looking at the current iteration here https://github.com/orgs/dotnet/projects/194/views/2?groupedBy%5BcolumnId%5D=Assignees it looks like we have about 10 people working on 25 issues in a timespan of 4 weeks.

Is this really all that Microsoft is investing in manpower into MAUI?

My first tries converting a large project from Xamarin to MAUI have been about 10 months ago. Today I come back to the same layout issues with basic stuff not updating, overlapping ... just simply not working. This is frustrating.

How much longer do I need to wait until this platform is usable for Android/iOS?

I was so hoping to be delighted coming back with many issues fixed and a stable platform.
I leave again with pain and sadness.

**Comment by manishbisen** [comment]
Interested in getting answer on this.

**Comment by HNordhaus** [comment]
We all hoped it gets better with MAUI, but Microsoft does not make a single cent with Xamarin/MAUI.
If I am right there are also no job openings for MAUI stuff.
So why should Microsoft invest in MAUI? Herioc people, e.g. Gerald, do their best, but is it too much for such a little crew.
I guess there are people who would like to work for Microsoft, me included, helping to improve MAUI, but I guess the budget does not allow it.
We are now at 2K issues, and Xamarin Forms is at 2.5K issues... The number goes up, not down....

But this is just my humble opinion
Henrik

**Comment by nextcodelab** [comment]
This is the problem with the native UI Link to each platform... maybe drawn control is easier to fix... but still support native view calling.
The team is too small to maintain all plaform specific codes,, that's team is enough for only drawn controls.. that easier to fix than native view that's lots of codes to maintain... (COMMUNITY ALSO BUSY FOR THEIR OWN JOBS FOR THE LIVING)_

I know .NET MAUI Team are heroes on the side of .net maui. they provide their best to make .net maui better.
Maybe microsoft thinks that they do not earn much on .Net Maui (BETTER .NET SUPPORTS ALSO MAKE MICROSOFT SERVICES MORE POPULAR), they do not give enough financials to it.. Unlike the flutter... Google give more finance to their own frameworks. MANY OF AZURE CUSTOMERS ARE ALSO FROM .NET sides.   => Better .Net supports will make microsoft services also more popular.

**Comment by DoubleDBE** [comment]
That's not the only problem.

There are many PR's that don't really address the root problems (think shell, support for the next version of .NET, etc...)

I agree that when certain functionality is provided, it should work without issues, but not at the cost of the basics.

I don't care about .NET 8 support while basic functionality doesn't work. Don't care much about shell either.

I'd rather stay with .NET 6 and have all basic controls "just work".

@davidortinau please take note, thanks!

I know the team is doing the best they can, But supporting the latest and greatest when core functionality isn't working (or has many issues) feels more like a marketing stunt than something else.

**Comment by akhanalcs** [comment]
I'm also interested to know an answer to this question from MSFT.

**Comment by HausBJB** [comment]
Hello, this is really strange what is going on here. The best example is something like this here https://github.com/dotnet/maui/issues/9485 how to move it to net8.

**Comment by mphill** [comment]
The problems are bigger than MAUI.  To be successful you also need tooling and an ecosystem that works as well.

- Visual Studio Mac is **loaded** with issues for MAUI and Xamarin devs.   Every new Xcode release seems to cause issues resulting in downtime or forced downgrades.
- Hot reload.  I'll just leave it there.
- AppCenter doesn't even support .net 6 let alone MAUI (let that sink in).  Bitrise, one of the last cloud CI/CD pipelines to support Xamarin is dropping support and MAUI support doesn't look promising. DevOps pipelines are not an appropriate alternative either.
